src/utilities/include/public.h

```diff
@@ -150,17 +150,36 @@ using Polygons = std::vector<SimplePolygon>;
 
 /**
  * An alternative to Mesh for output suitable for pushing into graphics
- * libraries directly.
+ * libraries directly. This may not be manifold since the verts are duplicated
+ * along property boundaries that do not match. The additional merge vectors
+ * store this missing information, allowing the manifold to be reconstructed.
```

Comment:
This is the core of the new API, but it's still incomplete. For now the mesh/material/triangle reference still lives in the `MeshRelation`; I'm planning to build a different version of that information here and probably deprecate `MeshRelation` entirely.